at-tanvo/Exercise2.rb

```diff
@@ -0,0 +1,13 @@
+def sum a
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/47f87600d63703e4dcdf4c1477df3685adc982eb/at-tanvo/Exercise2.rb#L1"></span>Naming/UncommunicativeMethodParamName: Method parameter must be at least 3 characters long.</td>
    </tr>
  </tbody>
</table>